src/pipeline/CommonBindings.cpp

```diff
@@ -108,6 +111,19 @@ void CommonBindings::bind(pybind11::module& m, void* pCallstack){
         .def_readwrite("height", &Rect::height)
         ;
 
+    stereoPair
+        .def(py::init<>())
+        .def_readwrite("left", &StereoPair::left)
+        .def_readwrite("right", &StereoPair::right)
+        .def_readwrite("baseline", &StereoPair::baseline)
+        .def_readwrite("isVertical", &StereoPair::isVertical)
+        .def("__repr__", [](StereoPair& stereoPair) {
```

Comment:
__str__ not needed?

Reply:
You mean` __str__'`in python?

If not specified `__repr__` is called instead by python.